src/Toolkit/registry/default/registry.json

```
@@ -0,0 +1,379 @@
+{
+    "name": "Symfony",
+    "licenses": [
+        "MIT"
+    ],
+    "homepage": "https://www.symfony.com",
+    "items": [
+        {
+            "name": "Alert",
+            "manifest": "components/Alert.json",
+            "theme": "",
+            "type": "component",
+            "dependencies": [
+                "Button"
+            ]
+        },
+        {
+            "name": "AspectRatio",
+            "manifest": "components/AspectRatio.json",
+            "theme": "",
+            "type": "component",
+            "dependencies": []
+        },
+        {
+            "name": "Avatar",
+            "manifest": "components/Avatar.json",
+            "theme": "",
+            "type": "component",
+            "dependencies": [
+                "AvatarFallback",
+                "AvatarImage"
+            ]
+        },
+        {
+            "name": "AvatarFallback",
+            "manifest": "components/Avatar/AvatarFallback.json",
```

Let's remove the prefix `Foo` when using `Foo:FooBar`, the repetition does not had plus-value, instead we should have `Foo:Bar` (e.g.: `Table:Cell`, `Grid:Col`, `Breadcrumb:Item`, etc...) 
```suggestion
            "manifest": "components/Avatar/Fallback.json",
```